sdk/java-sdk/src/main/java/com/akkaserverless/javasdk/GrpcStatus.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.javasdk;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A gRPC status code.
+ */
+public enum GrpcStatus {
```

Comment:
We could also use `io.grpc.Status.Code` rather than introducing our own type, but I'm fine with either.

Perhaps consider https://github.com/lightbend/akkaserverless-java-sdk/issues/830#issuecomment-1032837992

Reply:
Yeah, I did think about that. Are we guaranteed to always have `io.grpc.Status.Code` on the classpath? If `akka-grpc` switches to an `akka-http` based client implementation, it might go away right?

Reply:
Since `io.grpc.Status.Code` is part of `grpc-api` (and not `grpc-core`), I expect `akka-grpc` to keep depending on this even when we switch to `akka-http` for the transport implementation.

Reply:
Ok, I've switched to `io.grpc.Status.Code`.